de.dlr.sc.overtarget.language/src/de/dlr/sc/overtarget/language/TargetModel.xtext

```diff
@@ -60,7 +60,7 @@ SimpleVersion:
 ;
 
 Unit:
-	'Unit' source=Source 'version' vers= (Version | 'newest') ';'
+	{Unit} 'Unit' addAll?='addAll'? (source=Source 'version' vers= (Version | 'newest'))? ';'
```

Comment:
Wouln't it make more senese to have this addAll option as paort of the RepoLoc? rathern than the Unit? I mean technically a Unit cannot be all Units, right? Not sure, just asking...

Reply:
Yes, I think you're right. Thank you :)